pkg/codegen/testing/test/program_driver.go

```diff
@@ -104,6 +104,12 @@ var PulumiPulumiProgramTests = []ProgramTest{
 		Description: "Python regression test for #10914",
 		Skip:        allProgLanguages.Except("python"),
 	},
+	{
+		Directory:   "simplified-invokes",
+		Description: "Simplified invokes",
+		Skip:        allProgLanguages.Except("dotnet").Except("nodejs"),
```

Comment:
This reads easier to me.
```suggestion
		Skip:        codegen.NewStringSet("python", "go")
```

Reply:
Indeed